### ISSUE TITLE
Error loading timezone aliases: inStream parameter is null

### ISSUE THREAD

**Issue by rafafialho6** [ical4j/ical4j#721]
**Describe the bug**
We're facing some warnings after upgrading the version (from 3.2.14 to 4.0.2)

loggerClassName: org.slf4j.impl.Slf4jLogger
loggerName: net.fortuna.ical4j.model.TimeZoneRegistryImpl
message: Error loading timezone aliases: inStream parameter is null

**To Reproduce**
1. Using latest version (4.0.2)
2. Implemented the lib on a gradle project:
```
    // iCal
    implementation ('org.mnode.ical4j:ical4j:4.0.2') {
        exclude group: 'org.codehaus.groovy'
    }
```
On the first file it tries to process the warnings are generated. (not sure if we should be concerned or not, specially with iCal that includes a timezone)

We're building an uber-jar (which includes the dependencies).

**Environment (please complete the following information):**
 - OS: Ubuntu Server 24.04 LTS, happens on MacOS as well
 - Java Version 21
 - iCal4j Version 4.0.2

Thank you. Still doing some research and taking a look at the changelogs to see if there's something we need to do.

**Comment by benfortuna** [comment]
Hi, I think this is already reported here - #712

Fix will be in 4.0.3

**Comment by rafafialho6** [comment]
Thanks! Sorry for the dupe.
